api/graphql/schema.py

```diff
@@ -700,7 +792,7 @@ class Query:
     """GraphQL Queries"""
 
     @strawberry.field()
-    def enum(self, info: Info) -> GraphQLEnum:
+    def enum(self, info: Info) -> GraphQLEnum:  # type: ignore
```

Comment:
This could probably be an line comment, but GraphQLEnum is a function call, and python type inferencing doesn't like that, but strawberry needs it (because it resolves this at runtime)